boards/arm/efm32gg_slwstk6121a/doc/index.rst

```diff
@@ -0,0 +1,180 @@
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32gg_slwstk6121a/efm32gg_slwstk6121a_defconfig``
+
+Other hardware features, including the WF200 WiFi transceiver, are
+currently not supported by the port.
+
+Connections and IOs
+===================
+
+The WGM160P's EFM32GG11 SoC has six GPIO controllers (PORTA to PORTF), all of which are
+currently enabled for the SLWSTK6121A board.
+
+In the following table, the column **Name** contains pin names. For example, PE1
+means pin number 1 on PORTE, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PA4   | GPIO        | LED0                                |
++-------+-------------+-------------------------------------+
+| PA5   | GPIO        | LED1                                |
++-------+-------------+-------------------------------------+
+| PD6   | GPIO        | Push Button PB0                     |
++-------+-------------+-------------------------------------+
+| PD8   | GPIO        | Push Button PB1                     |
++-------+-------------+-------------------------------------+
+| PE7   | UART_TX     | UART TX Console VCOM_TX US0_TX #1   |
++-------+-------------+-------------------------------------+
+| PE6   | UART_RX     | UART RX Console VCOM_RX US0_RX #1   |
++-------+-------------+-------------------------------------+
+| PB11  | I2C_SDA     | SENSOR_I2C_SDA I2C1_SDA #1          |
++-------+-------------+-------------------------------------+
+| PB12  | I2C_SCL     | SENSOR_I2C_SCL I2C1_SCL #1          |
++-------+-------------+-------------------------------------+
+
+
+System Clock
+============
+
+The EFM32GG11 SoC is configured to use the 50 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32GG11 SoC has four USARTs, two UARTs and two Low Energy UARTs (LEUART).
+USART0 is connected to the board controller and is used for the console.
+
+Programming and Debugging
+*************************
+
+.. note::
+   Before using the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+The SLWSTK6121A includes an `J-Link`_ serial and debug adaptor built into the
+board. The adaptor provides:
+
+- A USB connection to the host computer
+- A physical UART connection which is relayed over interface USB serial port.
+
+Flashing an application to SLWSTK6121A
+--------------------------------------
+
+The sample application :ref:`hello_world` is used for this example.
+Build the Zephyr kernel and application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: efm32gg_slwstk6121a
+   :goals: flash
```

Comment:
Either this goal should be `build` instead of `flash` or in the above sentence flashing should be mentioned also.

Reply:
I updated the sentence before to indicate this example both builds and flashes.